tools/parse/src/main/scala/org/allenai/nlpstack/parse/poly/polyparser/ParseFile.scala

```diff
@@ -85,6 +85,11 @@ object ParseFile {
     * @param parseSource the parses we want to evaluate against
     */
   def evaluateParserOnTestSet(parser: TransitionParser, parseSource: PolytreeParseSource): Unit = {
+    println("Loading lazy data structures.")
```

Comment:
We should switch these `println` statements to use `log.info` -- I cut a ticket re this (https://github.com/allenai/nlpstack/issues/138).  

That way a consuming API could redirect these log messages to it's preferred log location (or silence them).